examples/drawing/plot_center_node.py

```diff
@@ -0,0 +1,21 @@
+"""
+====================
+Custom Node Position
+====================
+
+Draw a graph with node(s) located at user-defined positions.
+
+When a position is set by the user, the other nodes can still be neatly organised in a layout.
+"""
+
+import networkx as nx
+import numpy as np
+
+G = nx.path_graph(20)  # An example graph
+center_node = 5  # Or any other node to be in the center
+edge_nodes = set(G) - {center_node}
+pos = nx.circular_layout(
+    G.subgraph(edge_nodes)
+)  # Ensures the nodes around the circle are evenly distributed
+pos[center_node] = np.array([0, 0])  # Or off-center - whatever the user needs
```

Comment:
Again, just a nit - the original comment made more sense in the context of the PR discussion; this might read a bit better. NBD either way!

```suggestion
pos[center_node] = np.array([0, 0])  # manually specify node position
```